src/components/PanelResizer.js

```diff
@@ -12,7 +12,8 @@ class PanelResizer extends React.Component {
 
         this.state = {
             isUpDownResizing: false,
-            isLeftRightResizing: false
+            isLeftRightResizing: false,
```

Comment:
Maybe `Horizontal` and `Vertical` would be more suitable then `LeftRight` and `UpDown`?